### DISCUSSION TITLE
HIDDEN_AUTHORS

### DISCUSSION THREAD

**Discussion by varna9000** [getnikola/nikola#3528] [Q&A]
I want to hide author name from the post in boostrap4 theme. In ``conf.py`` I've got ``BLOG_AUTHOR = "admin"`` and ```HIDDEN_AUTHORS = ['admin'] ```, but the name still shows under post titles after re-build. Do I need to do something else?

**Comment by Kwpolska** [comment]
`HIDDEN_AUTHORS` affects the list-of-authors page. To remove authors from post headers, you’ll need to change the templates and remove the code related to it (`post_header.tmpl` in the default templates).

**Reply by varna9000** [reply]
Ah, ok. I thought I could do it without changing the template code. Thanks!

**Comment by sukiletxe** [comment]
I found the same thing, but didn't reply due to the documentation in the variable's comment. Is the documentation wrong?

**Reply by Kwpolska** [reply]
The docs were wrong, I just fixed them.
